extension.js

```diff
@@ -0,0 +1,37 @@
+// copied from https://github.com/vrjuliao/sml-vscode-extension
+// The module 'vscode' contains the VS Code extensibility API
+// Import the module and reference it with the alias vscode in your code below
+const vscode = require('vscode');
+const smlEnviron = require('./smlEnvironmentManager');
+// this method is called when your extension is activated
+// your extension is activated the very first time the command is executed
+
+
+/**
+ * @param {vscode.ExtensionContext} context
+ */
+function activate(context) {
+	// start sml exection
+	smlEnviron.start();
+	console.log('Congratulations, your extension "twelf" is now active!');
+	
+	// let execShortCode = vscode.commands.registerCommand('sml-environment.execShortCode', () => smlEnviron.execShortCode());
+	let restartRepl = vscode.commands.registerCommand('twelf-server.restart', () => smlEnviron.restart());
```

Comment:
Should this command be exposed to the user in the package.json contributes section? Right now i dont see where this command is called.